### DISCUSSION TITLE
Has anyone tried this plugin with obs 30.0?

### DISCUSSION THREAD

**Discussion by mdamaged** [DistroAV/DistroAV#924] [Q&A - Support]
Has anyone tried this plugin with obs 30.0? I do not want to update obs otherwise. Thanks!

**Comment by Koodikalle** [comment]
0 problems with win 10 at least. working just fine.

**Comment by jbcurler2010** [comment]
Further testing with 4.12.0 and 4.13.0 showing that it's working. May have misconfigured yesterday. Thanks.

**Comment by nettings** [comment]
Works for me with 30.0.2(?) package on Mac OS X and latest Git HEAD on Linux.

**Comment by Trouffman** [comment]
Works like a charm on Windows 10 & 11 with OBS 30.1.2 (tested all weekend)
